src/Features/Features.php

```diff
@@ -128,32 +128,40 @@ public static function load_features() {
 	}
 
 	/**
-	 * Check if a feature is enabled.  Defaults to true for all features unless they are optional.
+	 * Gets a build configured array of enabled WooCommerce Admin respecting optionally disabled features.
 	 *
-	 * @param string $feature Feature slug.
-	 * @return bool
+	 * @return array Enabled Woocommerce Admin features/sections.
 	 */
-	public static function is_enabled( $feature ) {
-		if ( ! self::exists( $feature ) ) {
-			return false;
-		}
+	public static function get_available_features() {
+		$features                      = self::get_features();
+		$optional_features_unavailable = [];
 
-		$features = self::get_optional_feature_options();
-
-		if ( isset( $features[ $feature ] ) ) {
-			$feature_option = $features[ $feature ];
-			// Check if the feature is currently being enabled.
-			/* phpcs:disable WordPress.Security.NonceVerification */
-			if ( isset( $_POST[ $feature_option ] ) && '1' === $_POST[ $feature_option ] ) {
```

Comment:
Why doesn't this logic need to be preserved?

Reply:
Whoops, good catch. I added back in https://github.com/woocommerce/woocommerce-admin/pull/7193/commits/4e8296c960461afe2994c35b5ff3de9086be0d76